content/handsOnReinforcementLearning/sheeprl/_index.en.md

```diff
@@ -0,0 +1,336 @@
+
+##### Define the Experiment
+The last thing to do is to define the experiment. You just need to define a `custom_exp.yaml` file in the `./sheeprl/configs/exp` folder and assemble the environment, the agent, and the other components of the SheepRL framework. In particular, there are four parameters that must be defined:
+1. `total_steps`: the total number of policy steps to compute during training (for more information, check <a href="https://github.com/Eclectic-Sheep/sheeprl/blob/main/howto/work_with_steps.md#policy-steps" target="_blank">here</a>).
+2. `buffer.size`: the dimension of the replay buffer.
+3. `cnn_keys`: the keys of frames in observations that must be encoded (and eventually reconstructed by the decoder).
+4. `mlp_keys`: the keys of vectors in observations that must be encoded (and eventually reconstructed by the decoder).
+
+Below is an example of an experiment config file.
+{{< github_code "https://raw.githubusercontent.com/diambra/agents/main/sheeprl/configs/exp/custom_exp.yaml" >}}
+
+{{% notice note %}}
+When defining the configurations of the experiment you can specify how frequently save checkpoints of the model, and if you want to save the final agent. For more information, check <a href="https://github.com/Eclectic-Sheep/sheeprl/blob/main/howto/logs_and_checkpoints.md#checkpointing" target="_blank">here</a>.
+{{% /notice %}}
+
+
+##### Train and Evaluate the Agent
+To run the experiment you just need to go into the `./sheeprl` folder and run the following command:
+```shell
+diambra run -s=2 python train.py exp=custom_exp
+```
+
+{{% notice note %}}
+You have to instantiate 2 docker containers because sheeprl automatically performs a test of the agent after training.
+{{% /notice %}}
+
+After training, you can decide to evaluate the agent as many times as you want. You can specify only a few parameters for evaluating your agent:
+1. The checkpoint of the agent that you want to evaluate (`checkpoint_path`, mandatory).
+2. The type of device on which you want to run the evaluation (`fabric.device`, default to `cpu`).
+3. Whether or not to capture the video of the evaluation (`env.capture_video`, default to `True`).
+
+The reason why only these three parameters need to be specified is to avoid inconsistencies, e.g. the checkpoint of one agent and the configurations of the evaluation refer to another one, or the model in the checkpoint has different dimensions from the model specified in the configurations. 
+This implies, however, that the evaluation script expects a certain directory structure. For this reason, the structure of the log directory should not be changed: all of it can be moved, but not the checkpoint individually, otherwise the script cannot automatically retrieve the environment and agent configurations.
+
+{{< github_code "https://raw.githubusercontent.com/Eclectic-Sheep/sheeprl/feature/split-p2e/sheeprl/configs/eval_config.yaml" >}}
```

Comment:
@michele-milesi my local compilation breaks here, saying that it was not possible to retrieve this file. Can you please double check that it is available?

Reply:
Sorry, I fixed the link. Thank you